### PR TITLE
zcmp: add page

### PR DESCRIPTION
**Pull request by gutjuri**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, I have some suggestions for this page.

References: [Manned](https://manned.org/zcmp.1) and [Arch Wiki](https://man.archlinux.org/man/zcmp.1.en)

File: pages/common/zcmp.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# zcmp
+
+> Invoke `cmp` on gzipped files.
+> More information: <https://www.gnu.org/software/gzip/manual/gzip.html>.
```
[the remark this answers is not in the document]
Comment (pixelcmtd) [reply pages/common/zcmp.md:4]: Isn't our unwritten rule to prefer https://manned.org/zcmp over https://manned.org/zcmp.1?

[the remark this answers is not in the document]
Comment (kbdharun) [reply pages/common/zcmp.md:4]: > Isn't our unwritten rule to prefer https://manned.org/zcmp over https://manned.org/zcmp.1?

Yeah, Thanks I will update my suggestion here.

File: pages/common/zcmp.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+- Invoke `cmp` on two files, uncompressing them if necessary:
+
+`zcmp {{path/to/file1.txt.gz}} {{path/to/file2.txt.gz}}`
+
+- Compare a file to its gzipped version (assuming that `path/to/file.txt.gz` exists):
```
[the remark this answers is not in the document]
Comment (gutjuri) [reply pages/common/zcmp.md:10]: I think its good to mention that the programme looks for a file that has the .gz extension. Hence, i'd rather not change this.

File: pages/common/zcmp.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# zcmp
+
+> Compare compressed files.
+> More information: <https://manned.org/zcmp>.
+
+- Invoke `cmp` on two files compressed via `gzip`:
+
+`zcmp {{path/to/file1.txt.gz}} {{path/to/file2.txt.gz}}`
```
Comment (kbdharun) [inline pages/common/zcmp.md:8]: Is the `.txt` extension required in this command?
```suggestion
`zcmp {{path/to/file1.gz}} {{path/to/file2.gz}}`
```

File: pages/common/zcmp.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+- Compare a file to its gzipped version (assuming that `path/to/file.txt.gz` exists):
+
+`zcmp {{path/to/file.txt}}`
```
Comment (kbdharun) [inline pages/common/zcmp.md:12]:
```suggestion
`zcmp {{path/to/file}}`
```
Same doubt here too , as the man page suggests that it works for any file

File: pages/common/zcmp.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kbdharun) [reply pages/common/zcmp.md:10]: Is this fine?
```suggestion
- Compare a file to its gzipped version (assuming `.gz` exists):
```

Or

```suggestion
- Compare a file to its gzipped version (assuming `.gz` exists already):
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**